hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/services/AlertsCriteria.java

```diff
@@ -68,6 +72,50 @@ public void setEndTime(Long endTime) {
         this.endTime = endTime;
     }
 
+    public Long getStartResolvedTime() {
+        return startResolvedTime;
+    }
+
+    /**
+     * @param startResolvedTime fetched Alerts must have resolvedTime greater than or equal to startResolvedTime
+     */
+    public void setStartResolvedTime(Long startResolvedTime) {
+        this.startResolvedTime = startResolvedTime;
+    }
+
+    public Long getEndResolvedTime() {
+        return endResolvedTime;
+    }
+
+    /**
+     * @param endResolvedTime fetched Alerts must have resolvedTime less than or equal to endResolvedTime
+     */
+    public void setEndResolvedTime(Long endResolvedTime) {
+        this.endResolvedTime = endResolvedTime;
+    }
+
+    public Long getStartAckTime() {
+        return startAckTime;
+    }
+
+    /**
+     * @param startAckTime fetched Alerts must have ackTime greater than or equal to startAckTime
+     */
+    public void setStartAckTime(Long startAckTime) {
```

Comment:
Ack is an optional status, we may need to add a note on what this returns if an alert went from open->resolved.  I guess this would not include that alert?